tests/phpunit/TestFramework/Coverage/ProxyTraceTest.php

```diff
@@ -178,7 +178,7 @@ public function test_it_exposes_its_test_locations(): void
         $this->assertCount(
             1,
             iterator_to_array($trace->getAllTestsForMutation(
-                new NodeLineRangeData(19, 22),
+                new NodeLineRangeData(19, 19),
                 true
```

Comment:
now - incorrect test. When the mutation is on the function signature, it must be 1-line range